src/java.base/share/classes/java/lang/String.java

```diff
@@ -784,26 +796,13 @@ static String newStringWithLatin1Bytes(byte[] src) {
      *
      * @throws CharacterCodingException for malformed input or unmappable characters
```

Comment:
Shouldn't we document NPE here too?
I mean there is NPE note in `private` method `newStringUTF8OrThrow`, but not in more widely opened `newStringOrThrow`